client/signup/steps/theme-selection/index.jsx

```diff
@@ -59,7 +48,8 @@ module.exports = React.createClass( {
 	},
 
 	getThemes() {
-		return this.props.signupDependencies.themes || this.props.themes;
```

Comment:
If you no longer want to support `props.themes`, you should remove it from the `propTypes` for this component.